grobid-core/src/main/java/org/grobid/core/engines/label/TaggingLabels.java

```diff
@@ -114,7 +130,7 @@
     public static final TaggingLabel EQUATION_MARKER = new TaggingLabelImpl(GrobidModels.FULLTEXT, EQUATION_MARKER_LABEL);
     public static final TaggingLabel PARAGRAPH = new TaggingLabelImpl(GrobidModels.FULLTEXT, PARAGRAPH_LABEL);
     public static final TaggingLabel ITEM = new TaggingLabelImpl(GrobidModels.FULLTEXT, ITEM_LABEL);
-    public static final TaggingLabel OTHER = new TaggingLabelImpl(GrobidModels.FULLTEXT, OTHER_LABEL);
+    public static final TaggingLabel OTHER = new TaggingLabelImpl(GrobidModels.FULLTEXT, GROBID_OTHER_LABEL);
```

Comment:
mmm I think we should keep `OTHER_LABEL` here because all these labels are in the Grobid realm. This is confusing to add GROBID specifically here because it cannot be something else than grobid labels.

Reply:
OK, indeed!